fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/info/ModifyTablePropertiesOp.java

```diff
@@ -371,6 +371,9 @@ public void validate(ConnectContext ctx) throws UserException {
             throw new AnalysisException("You can not modify property 'enable_unique_key_skip_bitmap_column'.");
         } else if (properties.containsKey(PropertyAnalyzer.PROPERTIES_STORAGE_PAGE_SIZE)) {
             throw new AnalysisException("You can not modify storage_page_size");
+        } else if (properties.containsKey(PropertyAnalyzer.PROPERTIES_STORAGE_MEDIUM)) {
+            this.needTableStable = false;
+            this.opType = AlterOpType.MODIFY_TABLE_PROPERTY;
```

Comment:
other properties's opType is MODIFY_TABLE_PROPERTY_SYNC, please check if MODIFY_TABLE_PROPERTY is correct